authentication/account/service/user_blackbox_test.go

```diff
@@ -81,13 +83,59 @@ func (s *userServiceBlackboxTestSuite) TestDeactivate() {
 		userToDeactivate := s.Graph.CreateUser()
 		token1 := s.Graph.CreateToken(userToDeactivate)
 		token2 := s.Graph.CreateToken(userToDeactivate)
+		githubTokenToRemove := s.Graph.CreateExternalToken(userToDeactivate, provider.GitHubProviderID)
+		openshiftTokenToRemove := s.Graph.CreateExternalToken(userToDeactivate, "33456e01-0ce4-4da2-b94d-daa968412662") // ID of the OpenShift cluster returned by gock on behalf of the cluster service
 		userToStayIntact := s.Graph.CreateUser()
 		token3 := s.Graph.CreateToken(userToStayIntact)
 		token4 := s.Graph.CreateToken(userToStayIntact)
+		githubTokenToKeep := s.Graph.CreateExternalToken(userToStayIntact, provider.GitHubProviderAlias)
+		openshiftTokenToKeep := s.Graph.CreateExternalToken(userToStayIntact, "02f2eee5-d01a-4119-9893-292a7d39b49e") // ID of the OpenShift cluster returned by gock on behalf of the cluster service
 
 		defer gock.OffAll()
+		// call to Cluster Service
+		gock.New("http://f8cluster").
+			Get("/api/clusters/auth").
+			// MatchHeader("Authorization", "Bearer "+saToken).
+			// MatchHeader("X-Request-Id", reqID).
```

Comment:
Why these checks are commented out?

Reply:
They don't match, but I'll remove them anyways because the focus of the test is not on how `auth` retrieves the list of clusters from the `cluster` service anyways

Reply:
removed in 658f033

Reply:
Fair point.